specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/virtualWan.json

```diff
@@ -0,0 +1,2105 @@
+      "description": "Result of the request to list VirtualWANs. It contains a list of VirtualWANs and a URL nextLink to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VirtualWAN"
+          },
+          "description": "List of VirtualWANs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any."
+        }
+      }
+    },
+    "VpnSiteProperties": {
+      "properties": {
+        "virtualWAN": {
+          "$ref": "./network.json#/definitions/SubResource",
+          "description": "The VirtualWAN to which the vpnSite belongs"
+        },
+        "deviceProperties": {
+          "description": "The device properties",
+          "$ref": "#/definitions/DeviceProperties"
+        },
+        "ipAddress": {
+          "type": "string",
+          "description": "The ip-address for the vpn-site."
+        },
+        "siteKey": {
+          "type": "string",
+          "description": "The key for vpn-site that can be used for connections."
+        },
+        "addressSpace": {
+          "$ref": "./virtualNetwork.json#/definitions/AddressSpace",
+          "description": "The AddressSpace that contains an array of IP address ranges."
+        },
+        "bgpProperties": {
+          "$ref": "./virtualNetworkGateway.json#/definitions/BgpSettings",
+          "description": "The set of bgp properties."
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "$ref": "#/definitions/ProvisioningState"
+        }
+      },
+      "description": "Parameters for VpnSite"
+    },
+    "VpnSite": {
+      "required": [
+        "location"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/VpnSiteProperties"
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "VpnSite Resource."
+    },
+    "ListVpnSitesResult": {
+      "description": "Result of the request to list VpnSites. It contains a list of VpnSites and a URL nextLink to get the next set of results.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VpnSite"
+          },
+          "description": "List of VpnSites."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any."
+        }
+      }
+    },
+    "GetVpnSitesConfigurationRequest": {
```

Comment:
great, and now there will be linked access check on the vpnSites passed in the body?

Reply:
Yes